docs/modules/ROOT/partials/vault-warning.adoc

```diff
@@ -0,0 +1,8 @@
+`component-vault` leverages k8up's application specific backups to create Vault snapshots.
+Restoring such a snapshot is only possible by providing the Vault unseal key and root token.
```

Comment:
I might link to the upstream docs here inline on what those tokens do. If i read smth like this my brain force me to google it 😅

- https://developer.hashicorp.com/vault/docs/concepts/tokens#root-tokens
- https://developer.hashicorp.com/vault/docs/concepts/seal